tests/intergration/song-read.test.js

```diff
@@ -0,0 +1,174 @@
+const { expect } = require('chai');
+const { getDb } = require('../../src/services/db');
+const { appGet } = require('../helpers/requestHelpers');
+const {
+  setupArtist,
+  setupAlbum,
+  setupSong,
+  tearDown,
+} = require('../helpers/setupHelpers');
+
+describe('read song', () => {
+  let db;
+  let artists;
+  let albums;
+  let songs;
+
+  beforeEach(async () => {
+    try {
+      db = await getDb();
+
+      await setupArtist(db, 3);
+      [artists] = await db.query('SELECT * from Artist');
+
+      
+      await setupAlbum(db, artists[0]);
+      await setupAlbum(db, artists[1]);
+      
+
```

Comment:
Could remove a blank line on either side of these `await` statements here, never really need more than one clear line at any point